test-apps/tutorial-servo-can-control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,26 @@
+#include "TutorialServo.h"
+#include "mbed.h"
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees, float minPulsewidthInMs, float maxPulsewidthInMs) {
+    this->m_servoPwmOut = new PwmOut(servoPin);
+    this->m_servoPwmOut->period_ms(20);  //setting the PWM period to 20ms
+    this->m_servoRangeInDegrees = servoRangeInDegrees;
+    this->m_minPulsewidthInMs = minPulsewidthInMs;
+    this->m_maxPulsewidthInMs = maxPulsewidthInMs;
+}
+
+void TutorialServo::setPositionInDegrees(const float degrees) {
+    this->m_servoPwmOut->pulsewidth((1 + degrees/this->m_servoRangeInDegrees) / 1000);
```

Comment:
I would remove `this->` since the `m_` prefix already tells us that that variable is a private member.

Reply:
Also, this mapping from degrees to ms isn't complete, do you need to consider `m_minPulsewidthInMs` and `m_maxPulsewidthInMs`?

Reply:
Also what if `degrees` was greater than the servo range or negative?